### PR TITLE
Feature/Implement service & controller methods to create a tour

### PR DESCRIPTION
**Pull request by Tony9984**
# Summary

Added service & controller methods to create a tour
P.S. Couldn't make up time to complete the Mapbox part... My bad, will investigate and get it done for next week. Could we maybe put it as a separate issue?

## Test Plan

Ran existing test cases

## Related Issues

#7 (Create a tour)

### CODE REVIEW

**Comment by JDziewonski98** [comment]
The code looks good, but I think you should also add some unit tests for the controller and service. We have no controller or service tests in the code yet, so it might take a little to figure out (i.e. mocking response from service in the controller tests) , but then we can use them as boilerplate for all the other controllers and services. The current tests are just basic persistence tests for the database model, so they do not prove the functionality of the controller and service.

**Comment by JDziewonski98** [comment]
Check this out for mocking the service within the controller tests: https://dev.to/jackcaldwell/mocking-es6-class-methods-with-jest-bd7

File: backend/src/api/routes.ts [hunk 1]
```diff
@@ -13,5 +15,9 @@ export class Routes {
       .route("/mural/:id")
       .get(this.muralController.show.bind(this.muralController))
       .put(this.muralController.update.bind(this.muralController));
+
+    app
+      .route("/route")
```
Comment (tedspare) [inline backend/src/api/routes.ts:20]: Should this be `"/tour"`?

File: backend/src/services/tour.service.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { Mural, MuralInterface } from "models/mural.model";
```
Comment (tedspare) [inline backend/src/services/tour.service.ts:1]: Is this import used?

**Review by tedspare — APPROVED** [review]
Added two minor comments otherwise LGTM

**Comment by Tony9984** [comment]
> The code looks good, but I think you should also add some unit tests for the controller and service. We have no controller or service tests in the code yet, so it might take a little to figure out (i.e. mocking response from service in the controller tests) , but then we can use them as boilerplate for all the other controllers and services. The current tests are just basic persistence tests for the database model, so they do not prove the functionality of the controller and service.

Ohhh sorry I assumed the existing tests were enough haha, got it then! I'll try and get something out tonight. Thanks for the link btw!

File: backend/src/api/routes.ts (hunk 1 above)
Comment (Tony9984) [reply backend/src/api/routes.ts:20]: Omg yeah mybad, will fix it ASAP. Thanks for pointing it out!

File: backend/src/services/tour.service.ts (hunk 2 above)
Comment (Tony9984) [reply backend/src/services/tour.service.ts:1]: Removed, thanks for the catch!

**Pull request merged**